domain/src/main/kotlin/de/busesteinkamp/domain/auth/AuthKey.kt

```diff
@@ -0,0 +1,6 @@
+package de.busesteinkamp.domain.auth
+
+import java.util.Date
+
+class AuthKey(val platformName: String, val key: String, val createdAt: Date, val expiresAt: Date) {
+}
```

Comment:
Das könnte auch eine data class sein meine ich

```suggestion
package de.busesteinkamp.domain.auth

import java.util.Date

data class AuthKey(
    val platformName: String, 
    val key: String, 
    val createdAt: Date, 
    val expiresAt: Date
)
```